### DISCUSSION TITLE
How does Avalonia operate SVG graphics Control for the Power industry？

### DISCUSSION THREAD

**Discussion by PanXiangquan** [wieslawsoltes/Svg.Skia#244] [Q&A]
![My](https://github.com/user-attachments/assets/1809aee6-cb86-4b0d-a2b3-1eadf8bdf808)

There are many SVG primitives in the interface, how do I register mouse events to manipulate these SVG primitives?

**Comment by wieslawsoltes** [comment]
https://wieslawsoltes.github.io/Svg.Skia/articles/guides/interaction-and-animation/



https://wieslawsoltes.github.io/Svg.Skia/articles/guides/hit-testing-and-model-editing/
